x-pack/plugins/security_solution/public/common/components/page_wrapper/index.tsx

```diff
@@ -16,12 +16,26 @@ import { useIsExperimentalFeatureEnabled } from '../../hooks/use_experimental_fe
 import { AppGlobalStyle } from '../page';
 
 const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+
+  > div {
```

Comment:
maybe something just like '.euiPageTemplate.kbnPageTemplate' ?

Reply:
could add whatever to x-pack/plugins/security_solution/public/app/home/template_wrapper/index.tsx StyledKibanaPageTemplate, but anything more specific is managed in packages/shared-ux/page/kibana_template/impl/src/page_template_inner.tsx

Reply:
actually looks like we can pass whatever, className to target this div, and mainProps.id to target the main, will add whatever id we want in place of the auto generated 'EuiPageTemplateInner_uuid' it is currently